lib/node_modules/@stdlib/stats/base/dists/gumbel/skewness/src/main.c

```diff
@@ -0,0 +1,67 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// MODULES //
+
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/sqrt.h"
+#include "stdlib/math/base/special/riemann_zeta.h"
+#include "stdlib/constants/float64/pi.h"
+
+
+// MAIN //
+
+/**
+* Returns the skewness for a Gumbel distribution with location `mu` and scale `beta`.
+*
+* @param {number} mu - location parameter
```

Comment:
Should not be JSDoc but Doxygen; see other C implementations for how the comments should look like.